### DISCUSSION TITLE
Tasmota app

### DISCUSSION THREAD

**Discussion by amirezz** [arendst/Tasmota#22532] [General]
Hello all my friends,

I am very proud to share with you my tasmota firmware devices control android app , it's free and fast and realable 

https://youtu.be/TuVg48aV8Vs?feature=shared

Here is the YouTube video for it and the code in the first comment,

Thanks for any feedback or support.

See you.

**Comment by Jason2866** [comment]
Without providing the source code it is not an application I would recommend to install the binary.
